src/main/java/pl/sda/giftwisher/giftwisher/GiftwisherApplication.java

```diff
@@ -1,13 +1,28 @@
 package pl.sda.giftwisher.giftwisher;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import pl.sda.giftwisher.giftwisher.gifts.model.dto.NewGiftDto;
+import pl.sda.giftwisher.giftwisher.gifts.service.GiftService;
 
 @SpringBootApplication
-public class GiftwisherApplication {
+public class GiftwisherApplication implements CommandLineRunner {
```

Comment:
Ciekawe. Jeżeli potrzebujecie tego tylko do wypełnienia listy losowymi danymi to można dorzucić taką logikę z innej strony, np blizej źródła danych.